src/app/zones/components/ZoneCreateKubernetesInstructions.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div>
+    <h3>1. {{ i18n.messages.zones.form.kubernetes.prerequisitesTitle }}</h3>
```

Comment:
yes, please, let's not only use t('path.to.key') , but let'
s consider to use `core/i18n` package https://github.com/Kong/public-ui-components/blob/main/packages/core/i18n/README.md and make it inline with other apps as well as the tools tech writers are using for defining those strings. And let's make this package better if need be instead gf having each app implement its own.

Reply:
hey @ValeryG 👋 

Quick TLDR on this one is, we will be using the `core/i18n` package very soon, in fact it cannot come sooner really!

Longer form: The plan as it was a few hours before the i18n stuff was added here was: In order to be able to merge this PR and then add `core/i18n` without one PR holding up the other, to follow the exact same interfaces in this PR as `core/i18n` i.e. `t('path.to.key')`. That way when the `core/i18n` comes in the next few PRs it will "Just Work"(tm). What we _don't want to do_ is start using object notation for i18n.

The locale files will in all likelihood be external also.

From the other comments you've written here (the markdown one is super interesting!) you and I (and Adam) seem to be on the exact same page 🙌

Reply:
I’ve changed the rudimentary i18n implementation to longer allow access to `i18n.messages` and used `i18n.t` throughout.